File: lista_01/02/MaiorNumero.java
import java.util.Scanner;

public class MaiorNumero {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        float maiorNumero = 0;
        int totalNumeros = 5;

        System.out.println("Insira " + totalNumeros + " números:");

        for (int i = 1; i <= totalNumeros; i++) {
            System.out.print("Número " + i + ": ");
            float numero = scanner.nextFloat();

            // Verifica se o número atual é maior que o maior número registrado
            if (numero > maiorNumero) {
                maiorNumero = numero;
            }
        }

        System.out.println("O maior número informado é: " + maiorNumero);
        scanner.close();
    }
}

File: lista_01/02/MaiorNumero.py
def encontrar_maior_numero():
    maior_numero = 0
    total_numeros = 5

    print(f"Insira {total_numeros} números:")

    for i in range(1, total_numeros + 1):
        while True:
            try:
                numero = float(input(f"Digite o número {i}: "))
                break
            except ValueError:
                print("Insira um valor válido de ponto flutuante.")

        if numero > maior_numero:
            maior_numero = numero

    print("O maior número é:", maior_numero)

if __name__ == "__main__":
    encontrar_maior_numero()
